float32     battery_voltage
float32     battery_current
float32     battery_capacitor_energy
uint8       battery_power_percent